website/src/components/Loading/LoadingScreen.jsx

```diff
@@ -1,12 +1,12 @@
-import { Progress } from "@chakra-ui/react";
+import { Container, Progress } from "@chakra-ui/react";
 
 export const LoadingScreen = ({ text }) => (
-  <div className="bg-slate-100">
+  <Container>
```

Comment:
If you're just replacing `div` elements without the most similar chakra element, i'd suggest `Box` instead of `Container`